02.calendar/cal.rb

```diff
@@ -22,12 +22,12 @@ def get_year_month
 end
 
 def print_cal(year, month)
-  first_day = Date.new(year, month, 1)
-  last_day = Date.new(year, month, -1)
-  print "".rjust(first_day.wday * 3) 
-  (first_day..last_day).each do |date|
+  first_date = Date.new(year, month, 1)
+  last_date = Date.new(year, month, -1)
+  print "   " * first_date.wday  
+  (first_date..last_date).each do |date|
     print "\n" if date.sunday? && date.day != 1 # 土曜日を表示した後に改行を入れる
-    print "#{date.day}".center(3)
+    print "#{date.day}".rjust(2) + " "
```

Comment:
```suggestion
    print date.day.to_s.rjust(2) + " "
```

to_s にした方が数値から文字列に変換することが明確にわかって良いです。